src/main/java/com/netflix/imflibrary/st0377/header/WaveAudioEssenceDescriptor.java

```diff
@@ -116,7 +51,7 @@ public int getQuantizationBits() throws MXFException
      */
     public int getBlockAlign() throws MXFException
     {
-        long value = this.waveAudioEssenceDescriptorBO.block_align;
```

Comment:
We can add a getWaveAudioEsenceDescriptorBO() method and use it in place of typecasting everywhere.

Reply:
Done.